src/main/java/clients/VehicleTechnicalRecordsClient.java

```diff
@@ -168,6 +193,82 @@ private Response callPutVehicleTechnicalRecordsWithAlterations(String vin, Strin
         return response;
     }
 
+    private Response callPutVehicleTechnicalRecordsForArchived(String systemNumber, String requestBody, List<JsonPathAlteration> alterations) {
```

Comment:
For every method that uses a list of json path alteration we should add at the end 'WithAlterations' so it is clear we are altering the request body before making the request, and also because there could be also a method doing the same thing without altering the request body before making the request

Reply:
Changes done.